res/functional-tests/configuration.toml

```diff
@@ -62,15 +62,14 @@ LogLevel = "INFO"
       PersistOnError = "false"
 
 [Service]
-BootTimeout = "30s"
-CheckInterval = "10s"
+HealthCheckInterval = "10s"
 Host = "localhost"
-ServerBindAddr = "" # if blank, uses default Go behavior https://golang.org/pkg/net/#Listen
 Port = 59705
-Protocol = "http"
-ReadMaxLimit = 100
-StartupMsg = "Configurable Application Service Started"
-Timeout = "5s"
+ServerBindAddr = "" # if blank, uses default Go behavior https://golang.org/pkg/net/#Listen
+StartupMsg = "app-function-tests Service Started"
```

Comment:
should the startup message more like  `app-functional-tests ....`?

Reply:
Not sure what your asking since that is what the message is... ;-)

Reply:
i saw you have function shouldn;t it be functional, the adjective?

Reply:
i guess i just tried to make more consistent with the directory since it is `res/functional-test/` . that's why i ask that question.

Reply:
Ahh, got it. Will adjust to be `app-functional-tests` since I am using the service keys for the `service` name.

Reply:
Fixed